lib/teleterm/daemon/gateway_cert_reissuer.go

```diff
@@ -109,19 +108,15 @@ func (r *GatewayCertReissuer) reissueCert(ctx context.Context, gateway *gateway.
 		return trace.Wrap(err)
 	}
 
-	clusterURI, err := uri.ParseClusterURI(gateway.TargetURI())
-	if err != nil {
-		return trace.Wrap(err)
-	}
```

Comment:
This fragment is exactly why I created the issue in the first place, so I'm happy to see it go. :D Like why does `reissueCert` have to return a potential error if `TargetURI` is malformed.